test/e2e/specs/editor/plugins/nonce.spec.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Nonce', () => {
+	// While using beforeEach/afterEach is suboptimal for multiple tests, they
+	// are used here to ensure that the nonce plugin doesn't interfere with API
+	// calls made in global before/after calls, which may perform API requests.
+	test.beforeEach( async ( { requestUtils } ) => {
+		await requestUtils.activatePlugin( 'gutenberg-test-plugin-nonce' );
+	} );
+	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.deactivatePlugin( 'gutenberg-test-plugin-nonce' );
+	} );
+
+	test( 'should refresh when expired', async ( { page, admin } ) => {
+		await admin.createNewPost();
+		await page.keyboard.press( 'Enter' );
+		// eslint-disable-next-line no-restricted-syntax
+		await page.waitForTimeout( 5000 );
```

Comment:
It's a shame that we have to explicitly wait for 5 seconds here and **block** the whole testing process to test this one case. I wonder if we could refactor this with API mocking in the future 🤔.